pims/tiff_stack.py

```diff
@@ -146,7 +146,10 @@ def _read_metadata(self, tiff):
             except UnicodeDecodeError:
                 md[key] = ''
             if key == 'DateTime':
-                md[key] = _tiff_datetime(md[key])
+                try:
+                    md[key] = _tiff_datetime(md[key])
+                except Exception:
```

Comment:
Can we be a more selective with the exceptions we suppress here?

It would be good at also issue a `warning.warning` (as I don't think we have threading proper logging through pims yet).

Reply:
Good suggestions 👍